Sources/Models/SubscriberState.swift

```diff
@@ -27,16 +32,26 @@ public enum SubscriberState {
     switch self {
     case .nonSubscriber:
       return nil
-    case let .owner(_, status, _):
+    case let .owner(_, status, _, _):
       return status
-    case let .teammate(status, _):
+    case let .teammate(status, _, _):
       return status
     }
   }
 
+  public var deactivated: Bool? {
+    switch self {
+    case .nonSubscriber:
+      return nil
+    case let .owner(_, _, _, deactivated):
+      return deactivated
+    case let .teammate(_, _, deactivated):
+      return deactivated
```

Comment:
could be this too right?

```suggestion
    case let .owner(_, _, _, deactivated),
             .teammate(_, _, deactivated):
      return deactivated
```